loopsPartTwo.js

```diff
@@ -221,8 +232,36 @@ console.log(compareArray(routineThree, ["situp", "situp", "situp"]));
 
 function calculateRoutineDifficulty(routine) {
   let difficulty = "";
-  // Your Code Here!
-  return difficulty;
+  let situpPointValue = 1;
+  let legraisePointValue = 2;
+  let pushupPointValue = 4;
+  let pullupPointValue = 10;
+  let totalPoints = 0;
+  for (index = 0; index < routine.length; index++) {
+    if (routine[index] === "situp") {
+      routine[index] = situpPointValue;
+      totalPoints = situpPointValue + totalPoints;
```

Comment:
you can use the `+=` operator on these to be a little cleaner:  `totalPoints += situpPointValue`